runtime/src/transaction_priority_details.rs

```diff
@@ -26,6 +26,7 @@ pub trait GetTransactionPriorityDetails {
                 true,  // use default units per instruction
                 false, // stop supporting prioritization by request_units_deprecated instruction
                 true,  // enable request heap frame instruction
+                true,  // enable support set accounts data size instruction
```

Comment:
Passing `true` here to get transaction's prioritization details before and after feature activation.